Client/Frontend/Browser/Onboarding/OnboardingNavigationController.swift

```diff
@@ -19,6 +22,12 @@ protocol OnboardingControllerDelegate: class {
     func onboardingCompleted(_ onboardingController: OnboardingNavigationController)
 }
 
+enum OnboardingViewAnimationID: Int {
+    case background = 1
+    case details = 2
```

Comment:
you don't need to declare `2,3...` if you have assigned `1` to first case, it's inferred
You can leave it if you find it more readable though